prow/config.yaml

```diff
@@ -94,6 +94,21 @@ tide:
     kubernetes/website: squash
   target_url: https://prow.k8s.io/tide.html
 
+presets:
+- labels:
+    preset: service-account
```

Comment:
this is a list, so I guess `- preset: service-account`

Reply:
other way, it's a map, the jobs have labels written as lists which is wrong. fixing